<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="refuse">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>99</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>99</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>刷新好友</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>46</y>
      <width>355</width>
      <height>399</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>355</width>
      <height>399</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>355</width>
      <height>399</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>233</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>233</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>233</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 222, 222);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 222, 222);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>95</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>95</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>95</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>北京时间:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>300</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>399</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 127);</string>
    </property>
    <property name="digitCount">
     <number>19</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
